ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot

```diff
@@ -95,6 +95,8 @@ Uninstall RHODS V2
     ...    oc delete operatorgroup --all -n redhat-ods-operator --ignore-not-found
     Should Be Equal As Integers	${return_code}	 0   msg=Error deleting operatorgroup
     ${return_code}    ${output}    Run And Return Rc And Output    oc delete ns -l opendatahub.io/generated-namespace --ignore-not-found
+    # TODO: the RHODS namespaces do not have the above label, deleting explicitely - is this ok?
```

Comment:
in which case you saw that these namespace are not having the same label because if any namespace created by rhods it should have that label

Reply:
@tarukumar It was the Managed cluster on AWS, installation type CLI, the latest RHODS RC.

Reply:
The v2 operator is not compatible with the Addon. If you attempt to test it, you will notice abnormal behavior. This is likely why you observed that it was not deleted.

Reply:
@tarukumar When I checked again on self-managed, the label is there as you expected. I am going to remove the explicit namespace deletion and test it again on both managed and self-managed.